Lib/typing.py

```diff
@@ -2864,18 +2863,7 @@ class body be required.
     syntax form works for Python 2.7 and 3.2+
```

Comment:
This final paragraph from the docstring can also be deleted; we don't care about <Python 3.6 these days 😄

Reply:
Thanks for the (insanely fast) review!
This is outside the scope of this PR but I just noticed that there is another mention of Python 3.5 & 3.6 here as well:

https://github.com/python/cpython/blob/2b7027d0b2ee2e102a24a0da27d01b8221f9351c/Lib/typing.py#L2730

Reply:
Good spot! And we can probably get rid of the "In Python >= 3.6" mention in the same docstring here, too:

https://github.com/python/cpython/blob/2b7027d0b2ee2e102a24a0da27d01b8221f9351c/Lib/typing.py#L2713

Want to open a separate issue/PR about it?

Reply:
I wonder if we need to deprecate the keyword syntax there too, if type checkers don't support it. That's for another issue though.

Reply:
> Want to open a separate issue/PR about it?

Sure, will do!